runbotics/runbotics-desktop/src/action/jira/jira.utils.ts

```diff
@@ -241,6 +247,59 @@ export const getJiraAllSprintTasks = async <T extends CloudJiraUser>(
     };
 };
 
+export const getJiraAllTaskDetails = async <T extends CloudJiraUser>(
+    input: GetTaskDetailsInput,
+) => {
+    const jql = `key="${input.task}"`.trim();
+
+    let startAt = 0;
+    let issues: SearchIssue<T> | null = null;
+
+    const response = await searchTaskDetails<T>(input, jql, startAt);
+    if (response.total > 0) {
+        issues = response.issues[0];
+    }
+
+    return {
+        issues,
+    };
+};
+
+const searchTaskDetails = async <T extends CloudJiraUser>(
+    input: GetSprintTasksInput,
+    jql: string,
+    startAt: Page['startAt'],
+)  => {
+    const { username, password, originUrl } = input;
+    const baseFields = 'summary, issuetype, created, resolutiondate, updated, status, assignee, issuelinks, priority, reporter, components, labels, sprint, epic link, timespent, duedate, customfield_10020, customfield_10014';
+    const additionalFields = input?.fields
+        ? input?.fields
+            .split(',')
+            .map(field => field.trim())
+            .filter(field => !baseFields.split(',').includes(field))
+            .join(',')
+        : '';
+
+    const fields = additionalFields
+        ? `${baseFields},${additionalFields}`
+        : baseFields;
+    const { data } =  await externalAxios.get<IssueWorklogResponse<T>>(
+        `${originUrl}/rest/api/2/search`,
+        {
+            params: {
+                jql,
+                maxResults: 100, // 100 is max value
```

Comment:
instead of comment I'd use `const MAX_SEARCH_RESULTS = 100` at the top of the function body

Reply:
that's because comments can get outdated much easier than the code